warn/scrapers/ny.py

```diff
@@ -125,5 +134,53 @@ def _get_historical_data(cache):
     return dict_list
 
 
+def _get_historical_pdf_data(cache):
+    # See https://github.com/biglocalnews/warn-scraper/issues/476
+    urls = (
+        "https://github.com/biglocalnews/warn-scraper/files/8400324/FL-22-0165.Records.for.Release_Part1.pdf",
+        "https://github.com/biglocalnews/warn-scraper/files/8400325/FL-22-0165.Records.for.Release_Part2.pdf",
+        "https://github.com/biglocalnews/warn-scraper/files/8400326/FL-22-0165.Records.for.Release_Part3.pdf",
+    )
+
+    # Fetch the given file from its URL or the cache, return the local path
+    def download(url):
+        filename = url.split("/")[-1]
+        return cache.download(f"ny/{filename}", url)

```

Comment:
Can we put in a catch in here where it won't download the PDF if it's already cached locally? It'll speed up the prod runs. You can see examples of this done [here](https://github.com/biglocalnews/warn-scraper/blob/main/warn/scrapers/sc.py#L75-L84).

Reply:
Ah, I was under the mistaken impression that `cache.download(...)` was already doing that checking behind the scenes. Can definitely add.